### DISCUSSION TITLE
Ray mailable should handle notifications

### DISCUSSION THREAD

**Discussion by xavi7th** [spatie/ray#123] [Ideas]
Great product. I think I am going to love using ray.

When I try to send a Notification to ray()->mailable() I get the following error
```
Argument 1 passed to Spatie\LaravelRay\Ray::mailable() must be an instance of Illuminate\Mail\Mailable, instance of App\Notifications\UserLoginNotification given
```
It would be nice if ray could present Notifications too as that is the major attraction for me to ray.

Thanks.

**Comment by freekmurze** [comment]
I'll add support for notifications soon!

**Reply by keizah7** [reply]
![image](https://user-images.githubusercontent.com/49227202/222200080-c07f1ba0-0745-4050-8ba3-e3d6cae0608b.png)
